src/UI/Buyer/src/app/config/app.config.ts

```diff
@@ -13,6 +13,8 @@ export const ocAppConfig: AppConfig = {
   marketplaceID: environment.marketplaceID,
   marketplaceName: environment.marketplaceName,
   translateBlobUrl: environment.translateBlobUrl,
+  supportedLanguages: environment.supportedLanguages,
+  defaultLanguage: environment.defaultLanguage,
```

Comment:
We should add these variables to docker as well, or at the very least add some sensible defaults in Buyer/src/assets/appConfigs and Seller/src/assets/appConfigs

Reply:
I had performed a quick search for docker variables, but didn't see any, but upon further review I see that only a subset of properties are included in docker. I assume we should consider updating the docker variables to be 1:1 with the buyer and seller configs?

For the buyer apps, I think it would be more optimised by making it configurable in the seller app against the API Client xp as it's already representing the storefront with the IsStorefront xp.

Reply:
> For the buyer apps, I think it would be more optimised by making it configurable in the seller app against the API Client xp

I'm not opposed to adding this as an enhancement and would let us do some cool things like changing themes on the fly but we'll need to consider how to map a buyer application to a specific API client. The easiest would be to have only the clientID stored as a frontend configuration. I've also thought about something that is more dynamic like mapping a domain to an API client. That requires a bit more set up the first time around but has some powerful implications. For example if someone has a wildcard domain they could theoretically pop up new buyers pretty much at will without any kind of deployment process.